src/commons/stores/cluster-store.js

```diff
@@ -218,10 +217,10 @@
             temp.name = (cluster.short_name && cluster.short_name !== "None") ? cluster.short_name : cluster.integration_id;
             temp.clusterId = cluster.cluster_id;
             temp.currentTask = cluster.current_job;
-            temp.jobType = JSON.parse(cluster.current_job).job_name;
-            temp.currentStatus = JSON.parse(cluster.current_job).status;
+            temp.jobType = cluster.current_job.job_name;
+            temp.currentStatus = cluster.current_job.status;
             temp.managed = cluster.is_managed === "yes" ? "Yes" : "No";
-            temp.currentTaskId = JSON.parse(cluster.current_job).job_id;
+            temp.currentTaskId = cluster.current_job.job_id;
```

Comment:
@shirshendu Ideally it should be cluster.current_job ? cluster.current_job.job_name : {}. As at the initial stage, cluster.current_job might be a empty object.

Reply:
I don't see the issue though. If `cluster.current_job` might be an empty object, all the attributes called on it will just be `null` anyway, and will be shown as blank on the UI.

Reply:
@shirshendu If the `current_job` property itself not present, then this will break. Its more useful for handling negative scenarios so that even if the `current_job` property doesn't exist, the cluster list can be rendered.

Reply:
Ack, handled.